core/src/main/java/com/linecorp/armeria/server/DefaultServerErrorHandler.java

```diff
@@ -103,4 +114,29 @@ private static boolean needsToWarn() {
         return Flags.annotatedServiceExceptionVerbosity() == ExceptionVerbosity.UNHANDLED &&
                logger.isWarnEnabled();
     }
+
+    @Nonnull
+    @Override
+    public AggregatedHttpResponse renderStatus(ServiceConfig config,
+                                               HttpStatus status,
+                                               @Nullable String description,
+                                               @Nullable Throwable cause) {
+        if (status.isContentAlwaysEmpty()) {
+            return AggregatedHttpResponse.of(ResponseHeaders.of(status));
+        }
+
+        final HttpData content;
+        try (TemporaryThreadLocals ttl = TemporaryThreadLocals.acquire()) {
+            final StringBuilder buf = ttl.stringBuilder();
+            buf.append("Status: ").append(status.codeAsText()).append('\n');
+            buf.append("Description: ").append(firstNonNull(description, status.reasonPhrase())).append('\n');
+            if (cause != null && config.verboseResponses() && !status.isSuccess()) {
+                buf.append("Stack trace:\n");
+                buf.append(Throwables.getStackTraceAsString(cause));
```

Comment:
I saw that we already have `Exceptions.traceText()`. Is it better to use our own version?